update explainer headline

marjisound opened the pull request:
<!-- In this repo you can label a PR with the "PR Deployment" label to deploy the code to a publicly accessible url -->
## What does this change?
Thsi PR updates the DCR headline for Explainer articles. 
- headline font weight is bold now
- The Explainer tag will be on top of headline
## Why?
Requested by editorial design team https://docs.google.com/document/d/1MqSldxfgR86kOJaXTFNaBs5LzPzSa7jC4D4YwqqMFwg/edit#
## Screenshots

| Before      | After      |
|-------------|------------|
| ![image](https://user-images.githubusercontent.com/15894063/185917701-76b067ce-4f4d-43ee-b662-5826c40bae49.png) | ![image](https://user-images.githubusercontent.com/15894063/185917739-1c96cb52-bb00-47b9-b996-801982141510.png) |

[before]: https://example.com/before.png
[after]: https://example.com/after.png

<!--
You can add extra rows by repeating the last row in the table and then using new unique labels. E.g.

| ![before2][] | ![after2][] |

You can then reference the labels and map them to corresponding links.

[before2]: https://example.com/before2.png
[after2]: https://example.com/after2.png
-->

oliverlloyd commented on dotcom-rendering/src/web/components/ArticleHeadline.tsx:
```diff
@@ -670,7 +670,7 @@ export const ArticleHeadline = ({
 										format.theme === ArticleSpecial.Labs
 											? labsFont
-											: standardFont,
+											: boldFont,
```
This is a change to Analysis here, was this intentional?

oliverlloyd replied:
Ah, it's already gone 👍

oliverlloyd commented on dotcom-rendering/src/web/components/ArticleHeadline.tsx:
```diff
@@ -689,6 +689,39 @@ export const ArticleHeadline = ({
 							)}
 						</div>
 					);
+				case ArticleDesign.Explainer:
```
Does this one need a new branch? Is the return code for feature/review etc. the same as this?

E.g. Could we do this:

```
switch (format.design) {
				case ArticleDesign.Review:
				case ArticleDesign.Recipe:
				case ArticleDesign.Explainer:
				case ArticleDesign.Feature:
					return (
						<div
```

This is just a guess, I could be wrong!

marjisound replied:
Great point, I wasn't sure which of the existing switch branches was appropriate for Explainer. This seems to be good. Just fixed it 👍 

The only point worth mentioning is that, for Analysis we are now having a different font style when theme is labs.  but now for Explainer, we don't apply labs font styles any more

oliverlloyd commented on dotcom-rendering/src/web/lib/decideDesign.ts:
```diff
@@ -56,6 +56,8 @@ export const decideDesign = ({
 			return ArticleDesign.FullPageInteractive;
 		case 'NewsletterSignupDesign':
 			return ArticleDesign.NewsletterSignup;
+		case 'ExplainerDesign':
```
Nice 👍

oliverlloyd reviewed (commented):
Now that we're adding a new design, should we also add new stories? Maybe in ArticleHeadline and in Layout?

🤔 For the layout stuff we'd need a new fixture added [in the gen-fixtures](https://github.com/guardian/dotcom-rendering/blob/feature-new-analysis-design/dotcom-rendering/scripts/test-data/gen-fixtures.js#L29) file. And then we can run `make gen-fixtures` and add it [to the Layouts.stories file here](https://github.com/guardian/dotcom-rendering/blob/feature-new-analysis-design/dotcom-rendering/src/web/layouts/Layout.stories.tsx#L32).

This will generate a lot of diffs but I think this is the right place to add this story

marjisound commented:
@oliverlloyd thanks so much for also explaining how to do the new layout story :) 

I added the new `Explainer` layout story following the steps you mentioned. Also added a new headline story for explainer.

oliverlloyd reviewed (approved):
Yep 👍
